CMakeLists.txt

```diff
@@ -455,6 +455,12 @@ if (HAVE_EXECINFO_H)
   set (HAVE_STACKTRACE 1)
 endif (HAVE_EXECINFO_H)
 
+if (WIN32)
+  set (HAVE_STACKTRACE 1)
+  set (HAVE_SYMBOLIZE 1)
+  target_link_libraries (glog PUBLIC Dbghelp.lib)
```

Comment:
I agree, and even more so, I think I should instead use:

```
#pragma comment(lib, "DbgHelp")
```

In an appropriate header, eliminating the need for us to `target_link_libraries` or for consumers of Glog to do so either.